packages/core/src/textures/GLTexture.ts

```diff
@@ -70,5 +76,7 @@ export class GLTexture
          * @member {number}
          */
         this.internalFormat = 5121;
+
+        this.samplerType = 0;
```

Comment:
I'm not sure why this is always `SAMPLER_TYPES.FLOAT`? Doesn't it depend on the internal format of the texture (if we should derive it from that?).

Reply:
This feature only for WebGL2 and it requires custom texture resource, we delegated all that stuff to user. However, user might not know how pixi batcher works,